tests/signup.js

```diff
@@ -15,15 +15,16 @@ limitations under the License.
 */
 
 const helpers = require('../helpers');
+const acceptTerms = require('./consent');
 const assert = require('assert');
 
-module.exports = async function signup(page, username, password, homeserver) {
+module.exports = async function signup(page, username, password, homeserver, options) {
   const consoleLogs = helpers.logConsole(page);
   const xhrLogs = helpers.logXHRRequests(page);
   await page.goto(helpers.riotUrl('/#/register'));
   //click 'Custom server' radio button
-  await page.waitForSelector('#advanced', {visible: true, timeout: 500});
-  await page.click('#advanced');
+  const advancedRadioButton = await helpers.waitAndQuerySelector(page, '#advanced');
```

Comment:
We're customising the HS but it looks like we set the default HS in the riot config? Both of these presumably aren't necessary? We should probably have tests for both signing up with & without a custom HS, although accepting the default will be the common case so that's probably what we ought to test first.

Reply:
Yeah, this code is from before having a local riot copy. But agreed on all points. We should test both but starting with the default makes sense, I'll change it :+1: